doc/man1/openssl-cmp.pod.in

```diff
@@ -1166,7 +1175,7 @@ In order to update the enrolled certificate one may call
 
   openssl cmp -section insta,kur
 
-using with PBM-based protection or
+using with MAC-based protection with PBM or
```

Comment:
The "with" here seems superfluous.

Reply:
Indeed the "M" in "PBM" overlaps with "MAC-based", but "PB" (password-based) does not -
at the latest with the advent of KEM-based message protection, other variants of MAC-based protection are possible, and we already plan to provide KEM-based protection for supporting PQC.

Reply:
I actually meant the first "with", i.e. "using MAC-based protection with PBM or ..." sounds better to me

Reply:
Oh, right, this was a clear mistake (though not part of the fixes done here, which is why I misunderstood your remark).
Fixed.